runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -10017,6 +10017,27 @@ static TR::Register *inlineSinglePrecisionFPTrg1Src2(TR::Node *node, TR::InstOpC
    return targetRegister;
    }
 
+static TR::Register *inlineSinglePrecisionFPTrg1Src3(TR::Node *node, TR::InstOpCode::Mnemonic op, TR::CodeGenerator *cg)
+   {
+   TR_ASSERT(node->getNumChildren() == 3, "Wrong number of children in inlineSinglePrecisionFPTrg1Src3");
```

Comment:
should be a fatal assert

Reply:
Yes.  Arguably, the others should be fatal asserts as well.

Reply:
Changed only my implementation from `TR_ASSERT` to `TR_ASSERT_FATAL` in https://github.com/eclipse/openj9/commit/66ab479e09f9e9987a63853c41547da3095bce75. The `assert` in other functions remains as just `TR_ASSERT`.